### PR TITLE
Extension marketplace blurb for schema create/evolve functionality.

### LINKED ISSUE #518: Update extension overview / readme regarding schema creation
Add a sentence or two about how to create new schemas in the extension overview / readme that shows up in the marketplace.

### PR DESCRIPTION
**Pull request by jlrobins**
## Summary of Changes

- Extension marketplace blurb for schema create/evolve functionality.

## Any additional details or context that should be provided?

- Closes #518

---

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +81,17 @@ and the user has the appropriate permissions.
 Schema definitions can be viewed by expanding the schema subject to see a specific schema version,
 then clicking the **View Schema** icon.
 
+##### Create / Evolve
+
+New or updated schemas and subject-bindings can be made simply by creating or opening an `.avsc`,
```
Comment (JimGalasyn) [inline README.md:86]:
```suggestion
You can create new schemas or update schemas and subject-bindings by creating or opening an `.avsc`,
```

File: README.md [hunk 2]
```diff
@@ -79,6 +81,17 @@ and the user has the appropriate permissions.
...
+`.proto`, or `.json` file, then using the **Cloud Upload** icon in the upper-right of the buffer
+titlebar to begin the process of selecting a schema registry and subject to bind to. You can either
+bind to new subjects or provide a revised/evolved schema for an existing subject to establish a new
+version.
+
+We recommend searching the marketplace for extensions to validate your Avro, JSON schema, or
```
Comment (JimGalasyn) [inline README.md:92]:
```suggestion
Search the marketplace for extensions to validate your Avro, JSON schema, or
```

**Review by JimGalasyn — APPROVED** [review]
LGTM, with a couple of suggestions.

**Comment by jlrobins** [comment]
Thanks @JimGalasyn , updated.

**Review by JimGalasyn — APPROVED** [review]
LGTM!

**Pull request merged**